l10n_th_address/__manifest__.py

```diff
@@ -32,6 +32,5 @@
     ],
     'installable': True,
     'auto_install': False,
+    'post_init_hook': '_update_data_noupdate',
```

Comment:
OTOH there are quite a few important MQT checks that are still "beta"/informative. This could be the case.